editor.planx.uk/src/pages/Preview/Questions.tsx

```diff
@@ -105,9 +105,7 @@ const Questions = ({ previewEnvironment }: QuestionsProps) => {
       setPreviewEnvironment(previewEnvironment);
       if (isStandalone) {
         NEW.getLocalFlow(sessionId).then((state) => {
-          if (state) {
-            resumeSession(state);
-          }
+          // session data is resumed by ./ResumePage.tsx
```

Comment:
I believe this is was the cause of the strange reconciliation behavior. The Question layout was resetting the state to the pre-reconciliation state. 

Instead, I think the ResumePage should be responsible for updating state (calling `resumeSession`) after reconciliation.